app/assets/stylesheets/category.css

```diff
@@ -0,0 +1,76 @@
+.c-container {
+  /* height: 100vh; */
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(248, 249, 248);
+  font-family: 'Lato', sans-serif;
+}
+
+.c-c {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(248, 249, 248);
+  font-family: 'Lato', sans-serif;
+  justify-content: space-between;
+}
+
+.categories {
+  display: flex;
+  flex-direction: column;
+  padding: 0 20px;
+  padding-top: 20px;
+  font-family: 'Lato', sans-serif;
+}
+
+.category {
+  width: 100%;
+  background-color: white;
+  box-shadow: 0 0 4px rgba(0, 0, 0, 0.2);
+  height: 5rem;
+  display: flex;
+  justify-content: space-between;
+  padding: 3px 5px;
+}
+
+.details {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.c-name {
+  height: 95%;
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+.icon {
+  width: 70px;
+  height: 70px;
+  background-color: red;
+}
+
+.new {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: green;
+  padding: 2% 0;
+  margin: 10px 20px 0 0;
+}
```

Comment:
- [ ] Well done for moving the button to the bottom of the page as suggested 👏 However, I noticed that due to the process, there is some margin on the right side of the button. Since the right margin was not present in the last review, I am making it as required⛏🤓 Please ensure that you follow the design guidelines while implementing any changes.

During last review | Current design
---| ---|
![image](https://github.com/isume295/Budget-app/assets/106119105/7953943a-11ec-4474-a2ef-f7d8a78c30db) | ![image](https://github.com/isume295/Budget-app/assets/106119105/482d66ac-3b3b-4e1a-94b1-b1cab62468df)


During last review | Current design
---| ---|
![image](https://github.com/isume295/Budget-app/assets/106119105/1f87e241-a4cf-45de-bca5-89e51334ce83) | ![image](https://github.com/isume295/Budget-app/assets/106119105/c53a72d2-d449-4923-98bc-49c08536f4ce)